src/Mapper.cpp

```diff
@@ -28,9 +29,11 @@ namespace sn
             case CNROM:
                 ret.reset(new MapperCNROM(cart));
                 break;
+            case AxROM:
+            ret.reset(new MapperAxROM(cart,mirroring_cb));
```

Comment:
This should have a `break;` statement. It works now only because the default is empty but it could be an issue later